independent-publisher-connectors/ORB Intelligence/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4731 @@
+                          "orb_num": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Orb Number",
+                            "example": "23248971"
+                          },
+                          "orb_nums": {
+                            "type": "array",
+                            "items": {
+                              "type": "integer",
+                              "format": "int32"
+                            },
+                            "description": "Secondary Orb Numbers",
+                            "example": [
+                              "23248971"
+                            ]
+                          },
+                          "name": {
+                            "type": "string",
+                            "description": "Company name",
+                            "example": "Orb Intelligence"
+                          },
+                          "parent_orb_num": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Orb Number of a parent company, if this company is a subsidiary",
+                            "example": "null"
+                          },
+                          "parent_name": {
+                            "type": "string",
+                            "description": "Name of a parent company.",
+                            "example": "null"
+                          },
+                          "ultimate_parent_orb_num": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Orb Number of the ultimate parent company.",
+                            "example": "23248971"
+                          },
+                          "ultimate_parent_name": {
+                            "type": "string",
+                            "description": "Name of the ultimate parent company.",
+                            "example": "null"
+                          },
+                          "subsidiaries_count": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Number of subsidiaries"
+                          },
+                          "branches_count": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Number of company branches/locations"
+                          },
+                          "names": {
+                            "type": "array",
+                            "items": {
+                              "type": "string"
+                            },
+                            "description": "Company's other names \u2014 tradestyles, DBA names, former names, abbreviations",

```

Comment:
@aaryan2134 Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.